src/js/components/Drop/stories/Tooltip.js

```diff
@@ -1,40 +1,188 @@
-              round={{ size: 'medium', corner: 'left' }}
-            >
-              tooltip contents
-            </Box>
-          </Drop>
-        )}
+    <Box>
+      <Button
+        ref={ref}
+        onMouseOver={() => setOver(true)}
+        onMouseLeave={() => setOver(false)}
+        onFocus={() => setOver(true)}
+        onBlur={() => setOver(false)}
+        plain
+        onClick={() => console.log(`Clicked me ${index}`)}
+      >
+        <Box pad={{ vertical: 'small' }} align="center">
+          {icon}
+        </Box>
+      </Button>
+      {ref.current && over && (
+        <Drop
+          align={{ left: 'right' }}
+          target={ref.current}
+          plain
+          trapFocus={false}
+        >
+          <Box pad="small" background="pink">
+            <Text color="white">{name}</Text>
+          </Box>
+        </Drop>
+      )}
+    </Box>
+  );
+};
+
+const SidebarDropButton = ({ icon, name }) => {
+  const [open, setOpen] = useState(false);
+  const [over, setOver] = useState(false);
+
+  const ref = useRef();
+
+  return (
+    <Box fill="horizontal">
+      <DropButton
+        ref={ref}
+        onMouseOver={() => setOver(true)}
+        onMouseLeave={() => setOver(false)}
+        onFocus={() => setOver(true)}
+        onBlur={() => setOver(false)}
+        open={open}
+        onOpen={() => setOpen(true)}
+        onClose={() => setOpen(false)}
+        dropAlign={{
+          top: 'top',
+          right: 'right',
+          left: 'right',
+          bottom: 'top',
+        }}
+        dropContent={<DropContent onClose={() => setOpen(false)} />}
+      >
+        <Box>{icon}</Box>
+      </DropButton>
+      {ref.current && over && (
+        <Drop
+          align={{ left: 'right' }}
+          target={ref.current}
+          plain
+          trapFocus={false}
+        >
+          <Box pad="small" background="pink">
+            <Text color="white">{name}</Text>
+          </Box>
+        </Drop>
+      )}
+    </Box>
+  );
+};
+const Sidebar = () => {
+  return (
+    <Box direction="row" fill>
+      <Box
+        direction="column"
+        basis="15%"
+        justify="between"
+        style={{ border: 'blue solid 4px' }}
+        align="center"
+      >
+        <Box>
+          <Nav>
+            {icons.map(({ icon, name }, index) => {
+              return (
+                <SidebarButton
+                  key={name}
+                  icon={icon}
+                  index={index}
+                  name={name}
+                />
+              );
+            })}
+            {menuIcons.map(({ icon, name }, index) => {
+              return (
+                <SidebarDropButton
+                  key={name}
+                  icon={icon}
+                  index={index}
+                  name={name}
+                />
+              );
+            })}
+          </Nav>
+        </Box>
+        <Box>
+          <SidebarButton icon={<Action />} index="Action" name="Action" />
```

Comment:
The last icon on theSidebar is looking funky
![image](https://user-images.githubusercontent.com/6320236/90573072-ba84ad00-e172-11ea-80a6-a6eccb2f8ac7.png)